schemas/logs/okta/tests/samples/systemlog_samples.jsonl

```diff
@@ -0,0 +1,100 @@
+{"severity":"INFO","transaction":{"type":"WEB","id":"Xqv35Wg7bT74OrlHVCM7JgAAFHM","detail":{}},"target":null,"actor":{"displayName":"John Doe","detailEntry":null,"id":"00u3mo2ci0RuzhCQe357","type":"User","alternateId":"foo@bar.com"},"client":{"userAgent":{"rawUserAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.129 Safari/537.36","os":"Mac OS X","browser":"CHROME"},"zone":"null","device":"Computer","id":null,"ipAddress":"1.1.1.1","geographicalContext":{"city":"Madrid","state":"Madrid","country":"Spain","postalCode":"28005","geolocation":{"lat":40.4143,"lon":-3.7016}}},"authenticationContext":{"authenticationProvider":null,"credentialProvider":null,"credentialType":null,"issuer":null,"interface":null,"authenticationStep":0,"externalSessionId":"102iCf9Im9wSEmaVMYZNbEOMA"},"published":"2020-05-01T10:20:22.243Z","version":"0","request":{"ipChain":[{"source":null,"ip":"1.1.1.1","geographicalContext":{"city":"Madrid","state":"Madrid","country":"Spain","postalCode":"28005","geolocation":{"lon":-3.7016,"lat":40.4143}},"version":"V4"}]},"displayMessage":"User login to Okta","eventType":"user.session.start","debugContext":{"debugData":{"deviceFingerprint":"d46b17d2d7119ece5a466d6e6d4285db","requestId":"Xqv35Wg7bT74OrlHVCM7JgAAFHM","requestUri":"/api/v1/authn","threatSuspected":"false","url":"/api/v1/authn?"}},"legacyEventType":"core.user_auth.login_success","securityContext":{"asNumber":null,"asOrg":null,"isp":null,"domain":null,"isProxy":null},"uuid":"5d730bf4-8b95-11ea-8cbe-bf2eaab2e9ef","outcome":{"result":"SUCCESS","reason":null}}
```

Comment:
Do we need this file now that all samples have been set as test inputs and transferred to `generated_tests.yml` ?

Reply:
I want to stuff samples in the repo as well. We will have sample-testing soon (shouldparse PR) and these would be useful to test for conflicts and for E2E testing.

Reply:
There `samples` dir was duplicated by mistake, this was fixed.